### DISCUSSION TITLE
.Net Maui Blazor Communication

### DISCUSSION THREAD

**Discussion by Revenge12** [dotnet/maui#14941] [Ideas]
I have a .NET Maui Blazor application, I am trying to do communication from my .NET Maui page to my Blazor page.

I am getting coordinates in my Maui page and sending it to a database local Sqlite database, but when I close the Maui page I want a method to be invoked on the .razor page.

This method will then find a value in the table on the Sqlite database and then display it on the .razor page.

The only way I was able to return a object was when I used popup. But the issue with popup is you cannot fill the screen and then the map display is very small.

Is there any way you can implement a Push/Pop method that can return a object back to the .razor page? Or can we let a .NET Maui page that pops invokes a method on the screen.
If there is such a method can you please share the method with examples :-)

Hope my request is clear.

Thank you,

**Comment by Xyncgas** [comment]
Idc push/pop, I am only applying general programming methods here : Have memory be accessable from blazor, write to the memory in MAUI, you can use unmanaged memory and native pointer and even COMs to do it

**Reply by Xyncgas** [reply]
My way of writing application is stylish, I've developed this way because I am the only one working on my project, so I did whatever I wanted. I can't help you to do the way you wanted, but I can help myself in this scenario by defining a static contractor that binds MAUI with razor with they initialize, building a message channel to allow them to talk with eachother, whether via bytes or via pre-defined methods or via dynamic access, I can do whatever I want, using my brain, this is the only take away that's important
